### DISCUSSION TITLE
IISCrypto hardening (TLS, Ciphers, Hashes, Cipher ect)

### DISCUSSION THREAD

**Discussion by roelmeijer** [0x6d69636b/windows_hardening#92] [Ideas]
Is it a idea to do also the IISCrypto (TLS, Ciphers, Hashes, Cipher ect) settings/check in the Kitty?

**Comment by lordfiSh** [comment]
You can do that with https://github.com/0x6d69636b/windows_hardening/blob/master/lists/finding_list_microsoft_windows_tls.csv

**Comment by 0x6d69636b** [comment]
Thanks @lordfiSh :) I'm not up-to-date with IISCrypto, is something missing in the TLS list?

**Comment by roelmeijer** [comment]
Thanks. This works fine!

Roel

Op ma 11 jul. 2022 om 19:33 schreef Bernhard ***@***.***>:
